spec/cob_index/macros/custom_spec.rb

```diff
@@ -771,8 +771,8 @@
               <datafield tag="PRT" ind1=" " ind2=" ">
                 <subfield code="a">bar</subfield>
                 <subfield code="b">https://sandbox01-na.alma.exlibrisgroup.com/view/uresolver/01TULI_INST/openurl?u.ignore_date_coverage=true&amp;rft.mms_id=991026913959703811</subfield>
-                <subfield code="f">Access full text online.</subfield>
```

Comment:
@ebtoner looking at this again, Should we not add a new case for when f["k"] is present since public_note (f["f"]) is still actually being extracted too?